README.md

```diff
@@ -1 +1,54 @@
-# TicTac
\ No newline at end of file
+# Tic Tac Toe
+> Project built using basic Ruby.
+  This project contains:
+  This is a console base Game build in ruby. And object oriented programing concepts is used in the project
+# What it does
+Two players needed to play thi game.
+
+## Built With
+- Ruby
+
+# Start the Game 
+-bin/main.rb is the executable file which will start the Game
+-ruby bin/main.rb
```

Comment:
Please add a space after the dash, it should be `- bin/main.rb is the ....` for markdown. Otherwise is not going to render properly: https://github.com/Tahirbhalli/TicTac/tree/feature#start-the-game

Reply:
done